plugins/chat/assets/javascripts/discourse/components/chat-full-page-header.gjs

```diff
@@ -38,13 +39,20 @@ export default class ChatFullPageHeader extends Component {
     });
   }
 
+  @action
```

Comment:
this is a minor bug I corrected while working on this, basically if you were clicking on the threads button in a channel head, it would propagate the mousemove to the underlying chat message under the channel header and it would show the message menu which wouldn't close when threads panel would show as there would be no mouseleave event of the chat message.